device-types/Delta/SG110-A.yaml

```diff
@@ -0,0 +1,51 @@
+---
+manufacturer: Delta
+model: SG110-A
+slug: delta-sg110-a
+part_number: SG110-A
+u_height: 3
+is_full_depth: true
+airflow: front-to-rear
+subdevice_role: parent
```

Comment:
Please remove this if you change the `device-bays` to `module-bays`

Reply:
converted power supplies to modules, and removed related keys for parent/child devices